steamtinkerlaunch

```diff
@@ -17620,6 +17626,16 @@ function dlMod2nexurl {
 			writelog "INFO" "${FUNCNAME[0]} - Starting portable '$RUNPROTON run ${MO}.exe $1'"
 			STEAM_COMPAT_CLIENT_INSTALL_PATH="$STEAM_COMPAT_CLIENT_INSTALL_PATH" STEAM_COMPAT_DATA_PATH="$STEAM_COMPAT_DATA_PATH" "$RUNPROTON" run "${MO}.exe" "$1" 2>&1 | tee /tmp/RUNMO2DL.log
 			cd - >/dev/null || return
+		elif [ -d "${GMO2EXE%/*}" ] && [ -n "$MO2PFX" ] && [ -n "$MO2WINE" ]; then
+			writelog "INFO" "${FUNCNAME[0]} - Download Url '$1' for game '$MONGURL' using data from portable '$MYPORTDLDAT'"
+			cd "${GMO2EXE%/*}" >/dev/null || return
+			writelog "INFO" "${FUNCNAME[0]} - Starting portable MO2 in prefix '$MO2PFX' using '$MO2WINE'"
+			if [ "$MO2INST" ]; then
```

Comment:
I think this check has a mistake, although ShellCheck interestingly isn't flagging it up.

```suggestion
			if [ -n "$MO2INST" ]; then
```

Let me know if this sounds reasonable :-)

Reply:
Yup. I think both work, but -n is definitely the proper way. Pushed a fix.